packages/@expo/cli/src/start/server/middleware/ReactDevToolsPageMiddleware.ts

```diff
@@ -1,37 +0,0 @@
-import assert from 'assert';
-import path from 'path';
-import resolveFrom from 'resolve-from';
-import send from 'send';
-
-import { ExpoMiddleware } from './ExpoMiddleware';
-import { ServerRequest, ServerResponse } from './server.types';
-
-export const ReactDevToolsEndpoint = '/_expo/react-devtools';
-
-export class ReactDevToolsPageMiddleware extends ExpoMiddleware {
-  constructor(projectRoot: string) {
-    super(projectRoot, [ReactDevToolsEndpoint]);
-  }
-
-  override shouldHandleRequest(req: ServerRequest): boolean {
-    if (!req.url?.startsWith(ReactDevToolsEndpoint)) {
-      return false;
-    }
-    return true;
-  }
-
-  async handleRequestAsync(req: ServerRequest, res: ServerResponse): Promise<void> {
-    assert(req.headers.host, 'Request headers must include host');
-    const { pathname } = new URL(req.url ?? '/', `http://${req.headers.host}`);
-    const requestPath = pathname.substring(ReactDevToolsEndpoint.length) || '/';
-
-    const entryPath =
-      // Production: This will resolve when installed in the project.
-      resolveFrom.silent(this.projectRoot, 'expo/static/react-devtools-page/index.html') ??
```

Comment:
Oh interesting, did `expo/static/react-devtools-page/index.html` ever resolve? I don't think it did for modern package managers 🙈

Reply:
you are right! i'm going to blame the guy who wrote the strange code 🙈